### ISSUE TITLE
v0.5.0 broke "use_old_network_legend" on Arch Linux

### ISSUE THREAD

**Issue by aofei** [ClementTsang/bottom#316]
Starting `btm` with `use_old_network_legend = true` will cause the terminal to get stuck. And the `btm` process occupies 100% of one of my CPU cores.

My environment:

```zsh
$ uname -a
Linux as-server 5.9.8-arch1-1 #1 SMP PREEMPT Tue, 10 Nov 2020 22:44:11 +0000 x86_64 GNU/Linux
$ btm -V
bottom 0.5.0
```

**Comment by ClementTsang** [comment]
Can replicate, looking into fixing this now.

**Comment by ClementTsang** [comment]
Should be resolved by e04ce4fa1b42e99f00cf8825bcd58da43552214e.  Will look into pushing this today.
